### PR TITLE
Add actionbar into graph

### PR DESCRIPTION
**Pull request by SerhatSurguvec**
**Issues to solve**
#23

### CODE REVIEW

File: app/src/main/java/com/aconno/sensorics/dagger/mainactivity/MainActivityModule.kt [hunk 1]
```diff
@@ -220,4 +222,31 @@ class MainActivityModule {
...
+        mainActivity.application
+    )
+
+    @Provides
+    @MainActivityScope
+    fun provideLiveGraphViewModel(
+        liveGraphViewModelFactory: LiveGraphViewModelFactory,
+        mainActivity: MainActivity
+    ) =
+        ViewModelProviders.of(
```
Comment (borna-mdn) [inline app/src/main/java/com/aconno/sensorics/dagger/mainactivity/MainActivityModule.kt]: Reformat. Move this one line up.

File: app/src/main/java/com/aconno/sensorics/ui/livegraph/LiveGraphFragment.kt [hunk 2]
```diff
@@ -1,44 +1,49 @@
...
+    private lateinit var macAddress: String
+    private lateinit var graphName: String
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
-        if (sharedPreferences.getBoolean("keep_screen_on", false)) {
-            window.addFlags(WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON)
+        if (arguments == null) {
+            throw NullPointerException("Arguments cannot be null")
```
Comment (borna-mdn) [inline app/src/main/java/com/aconno/sensorics/ui/livegraph/LiveGraphFragment.kt]: Use IllegalArgumentException

File: app/src/main/java/com/aconno/sensorics/dagger/mainactivity/MainActivityModule.kt (hunk 1 above)
Comment (srishtigaihre) [reply app/src/main/java/com/aconno/sensorics/dagger/mainactivity/MainActivityModule.kt]: Code reformatted

File: app/src/main/java/com/aconno/sensorics/ui/livegraph/LiveGraphFragment.kt (hunk 2 above)
Comment (srishtigaihre) [reply app/src/main/java/com/aconno/sensorics/ui/livegraph/LiveGraphFragment.kt]: Exception name changed

**Pull request merged**